app/views/shared/_stats.html.erb

```diff
@@ -0,0 +1,15 @@
+<% @user ||= current_user %>
```

Comment:
||= nghĩa là gì

Reply:
Cái này toán tử bằng hoặc bằng nghĩa là lấy @user hoặc @user = current_user